docs/developers/tutorials/staking-contract.md

```diff
@@ -658,14 +614,12 @@ Now that we've added the unstake function, let's test it out on devnet. Build yo
 ```bash
 UNSTAKE_AMOUNT=500000000000000000
 
-unstake() {
-    mxpy --verbose contract call ${SC_ADDRESS} \
-    --proxy=${PROXY} --chain=${CHAIN_ID} \
-    --send --recall-nonce --pem=${USER_PEM} \
+mxpy --verbose contract call ${SC_ADDRESS} \
```

Comment:
Use the hardcoded address, but this time we can omit the phrase with "make sure to adjust ...".

Reply:
same here, I think is better to let the user go through all steps